src/V3Partition.cpp

```diff
@@ -2029,7 +2029,18 @@ class ThreadSchedule final {
     std::unordered_map<const ExecMTask*, uint32_t> threadId;
```

Comment:
At this point the thread ID is duplicated in both `threadId` and `mtaskState`. Suggest remove the `threadId` map.